cropduster/fields.py

```diff
@@ -28,13 +29,62 @@ def sizes(self):
         else:
             return self.field.db_field.sizes
 
-    def regenerate_crops(self):
+    def _get_new_crop_thumb(self, size):
+        # "Imports"
+        Image = self.field.db_field.rel.to
+        Thumb = Image._meta.get_field("thumbs").rel.to
+
+        box = Box(0, 0, self.width, self.height)
+        crop_box = Crop(box, self.path)
+
+        best_fit = size.fit_to_crop(crop_box, original_image=self.path)
+        fit_box = best_fit.box
+        crop_thumb = Thumb(**{
+            "name": size.name,
+            "width": fit_box.w,
+            "height": fit_box.h,
+            "crop_x": fit_box.x1,
+            "crop_y": fit_box.y1,
+            "crop_w": fit_box.w,
+            "crop_h": fit_box.h,
+        })
```

Comment:
This isn't the hill I want to die on. But I do think that using a dict to make passing a large number of kwargs more readable is reasonable and should be allowed.